### ISSUE TITLE
Broken Link to Documentation and last commit was 2017, is this repo still maintained?

### ISSUE THREAD

**Issue by QueerCodingGirl** [adfinis-sygroup/pyaptly#41]
The link to the documentation in the readme leads to a gitlab-pages 404 page.

Is this repo still maintained by the adfinis sygroup? (old issues & PRs, last commit 2017)
And if you do not want to maintain it, could you release it into the hands of someone who would like to maintain it, please?

**Comment by tongpu** [comment]
I believe @luk43 and @karras should be able to answer the question if we're still actively using pyaptly.

**Comment by QueerCodingGirl** [comment]
@tongpu Could you reach out to one of them and ask them directly, please?

**Comment by tongpu** [comment]
I've got some feedback from our team and we're still using it in production up to this date, but haven't made any changes to it in ages as it's still working fine for our case up to now.

Is there anything specific that you're missing right now or would want to contribute?

**Comment by QueerCodingGirl** [comment]
Yes, while i now think that i  have figured out how to write the config yml it would be nice if you could fix the generation and hosting of the documentation.

My biggest problem right now is that you do not seem to build this package for newer distro versions. I need an Ubuntu focal package. You have some other Ubuntu focal packages hosted on your deb repository ( http://aptly.adfinis-sygroup.ch/adsy-public/ubuntu ), but not the pyaptly package.

It really seems that the main problem is the build pipeline. If you could fix that, as i cannot do that for you, that would be great and fix at least these 2 problems.

This project uses another (already archived) project of yours for your build pipeline: https://github.com/adfinis-sygroup/pyproject/blob/8471700f2009599d037ba2b941aef704655b13f8/HOW_TO_RELEASE.rst
I guess this needs to be switched out for whatever you currently use to build and deploy.

**Comment by QueerCodingGirl** [comment]
@tongpu ^

**Comment by tongpu** [comment]
Please be aware that this will probably take a while and that I won't be the one implementing it.

We probably need to find a replacement for the `freeze` dependency and will also need to replace `pyproject`. I'll try to at least move the docs over to GitHub pages to solve that issue.

**Comment by ganwell** [comment]
We updated pyaptly for 2024. It is now working on recent systems again. There are [alpha/beta packages](https://github.com/adfinis/pyaptly/actions/runs/8279534476) on each merge build, please search for the lastest one. There is still a lot of love that the project could receive, for example we have deferred updating sphinx-doc and therefore there is currently no readdocs.
